src/odemis/util/raster.py

```diff
@@ -142,3 +142,45 @@ def point_in_polygon(p, polygon):
         p1x, p1y = p2x, p2y
 
     return inside
+
+
+def get_possible_intersections(row_pairs: List[Tuple], col_pairs: List[Tuple], include_neighbors: bool = True) -> Set[Tuple]:
```

Comment:
Looking at the code, `include_neighbors` essentially draws with a pixel size of 3 pixels instead of 1. Why making it the default? I think it's just because your code uses this parameter as True. However, from a generic point of view, I think it'd be more logical to have it `False` by default.